examples/map/masking_hmi.py

```diff
@@ -30,13 +30,18 @@
 )
 
 ###############################################################################
-# Next we call the :meth:`~sunpy.map.GenericMap.reproject_to` to reproject the HMI Map
-# to have exactly the same grid as the AIA Map.
-# :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_align_aia_hmi.py` provides more reference.
+# Next we create the HMI map and crop it to the same field of view as the AIA image.
 
 hmi = sunpy.map.Map(HMI_LOS_IMAGE)
-hmi = hmi.reproject_to(aia.wcs)
-hmi.nickname = 'HMI magnetogram'
+hmi = hmi.submap(aia.bottom_left_coord, top_right=aia.top_right_coord)
+
+###############################################################################
+# We then call the :meth:`~sunpy.map.GenericMap.reproject_to` to reproject the AIA Map
+# to have exactly the same grid as the HMI Map.
+# :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_align_aia_hmi.py` provides more reference.
```

Comment:
I maintain that this choice is really subtle and important and not calling it out here at all is doing the reader of this example a disservice.